packages/ove-app-charts/README.md

```diff
@@ -0,0 +1,55 @@
+# Charts App
+
+This app supports visualisation of charts (excluding network diagrams) using the OVE framework. It is based on [Vega](https://vega.github.io/vega/), a declarative language for creating, saving, and sharing interactive visualization designs. Vega supports Canvas and SVG rendering.
```

Comment:
"It is based on [Vega-Embed](https://github.com/vega/vega-embed), which provides support for both [Vega](https://vega.github.io/vega/) and [Vega-Lite](https://vega.github.io/vega-lite/): these are both decalrative languages for interactive visualizations, but Vega-Lite is a higher level language."